SmartDeviceLink/SDLMenuCell.m

```diff
@@ -27,7 +27,7 @@ - (instancetype)initWithTitle:(NSString *)title icon:(nullable SDLArtwork *)icon
 
     _title = title;
     _icon = icon;
-    _voiceCommands = voiceCommands;
+    _voiceCommands = ([voiceCommands count] == 0) ? nil : voiceCommands;
```

Comment:
I think it would be better to do this in `SDLMenuManager sdl_commandForMenuCell:withArtwork:position`. That way if the developer sets an empty array and later assumes it won't be nil, they won't be surprised. That's very much an edge case, but I think it's better to handle this awkward case internally rather than externally facing to the developer.

Also dot-syntax is preferred for property accessors, so `[voiceCommands count]` can just be `voiceCommands.count`.